cal/cal/static/js/app.js

```diff
@@ -352,7 +359,6 @@ function CategoryListCtrl($scope, $http, CalendarFilterService, CategoryService)
     CategoryService.editCategory(categoryId, category.newLabel)
       .then(function(returnedCategory) {
         category.label = returnedCategory.label;
-        category.hours = returnedCategory.hours;
```

Comment:
Does this work on your computer? When I click edit then click save, it still gives me cumulative. Also when I add keywords it gives me cumulative. Try all edge cases and let me know if they all work for you.